apps/masterbots.ai/components/routes/chat/chat.tsx

```
@@ -215,94 +246,69 @@ export function Chat({
+      try {
+        // * Loading: processing your request + opening pop-up...
+        setLoadingState("processing");
+        setIsOpenPopup(true);
+
+        await tunningUserContent(userMessage);
+      } catch (error) {
+        console.error("Error processing user message. Using og message. Error: ", error);
+      } finally {
+        setIsNewResponse(true);
+
+        return await appendNewMessage(userMessage)
```

Biome

[error] Unsafe usage of 'return'.

'return' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)